src/mtcp/mtcp_restart.c

```diff
@@ -67,6 +67,12 @@
 #define HUGEPAGES
 #define GNI
 
+/*TODO: This macro is used for CentOS specific changes for now.
+ * Later, we want to merge the code that works both for CentOS and Cori. 
+ */
+// if we define both GNI and CENTOS, assume we are running on CentOS
+#define CENTOS
```

Comment:
Line 68, I think we shouldn't have `GNI` when building on centos.

Reply:
hmm, what do you think @JainTwinkle?